Lib/test/test_typing.py

```diff
@@ -22,6 +22,7 @@
 from typing import IO, TextIO, BinaryIO
 from typing import Pattern, Match
 import abc
+from _abc import _reset_registry, _reset_caches
```

Comment:
What if _abc is not there (PyPy?)

IMO you should just rename `C._abc_registry.clear` to `C. _abc_registry_clear` and make sure it works when `_abc` is not available.

Reply:
Good point! Will fix this.